### PR TITLE
APR-003:added scroll and received data from the backend

### PR DESCRIPTION
**Pull request by Anton0512**

### CODE REVIEW

File: project-01/src/pages/todo-list-api/todo-list.page.component.api.jsx [hunk 1]
```diff
@@ -3,17 +3,80 @@ import {
...
+  const handleClick = () => setFetching(true);
+
+  console.log(photos);
 
   return (
     <STodoListApiContainer>
       <STitle>Todo List Api</STitle>
       <SButton onClick={handleGoBack}>Go Back</SButton>
+      <SButton onClick={handleClick}>page + 1</SButton>
+      {/* {state && ( */}
```
Comment (wsergii) [inline project-01/src/pages/todo-list-api/todo-list.page.component.api.jsx]: Remove comments

File: project-01/src/pages/todo-list-api/todo-list.page.component.api.jsx [hunk 2]
```diff
@@ -3,17 +3,80 @@ import {
...
+      <SPhotoContainer>
+        {photos.map((photo) => (
+          <SPhotoWrapper key={photo.id}>
+            <SPhotoImage src={photo.thumbnailUrl} />
+            <SPhotoTitle>{photo.title}</SPhotoTitle>
+          </SPhotoWrapper>
+        ))}
+      </SPhotoContainer>
+      {/* )} */}
```
Comment (wsergii) [inline project-01/src/pages/todo-list-api/todo-list.page.component.api.jsx]: Remove comments

File: project-01/src/pages/todo-list-api/todo-list.page.component.api.jsx (hunk 1 above)
Comment (Anton0512) [reply project-01/src/pages/todo-list-api/todo-list.page.component.api.jsx]: fixed

File: project-01/src/pages/todo-list-api/todo-list.page.component.api.jsx (hunk 2 above)
Comment (Anton0512) [reply project-01/src/pages/todo-list-api/todo-list.page.component.api.jsx]: fixed

**Review by wsergii — APPROVED** [review]

**Pull request merged**